packages/curve-ui-kit/src/widgets/Header/Header.tsx

```diff
@@ -1,19 +1,23 @@
 import { usePathname } from 'next/navigation'
 import { useMemo } from 'react'
-import { useTheme } from '@mui/material/styles'
+import Box from '@mui/material/Box'
 import { WalletToast } from '@ui-kit/features/connect-wallet'
 import { WagmiConnectModal } from '@ui-kit/features/connect-wallet/ui/WagmiConnectModal'
 import { useLayoutStore } from '@ui-kit/features/layout'
 import { useIsDesktop } from '@ui-kit/hooks/useBreakpoints'
 import { useBetaFlag } from '@ui-kit/hooks/useLocalStorage'
 import { isChinese, t } from '@ui-kit/lib/i18n'
 import { type AppName, getInternalUrl, PAGE_DISCLAIMER, PAGE_INTEGRATIONS, routeToPage } from '@ui-kit/shared/routes'
-import { DESKTOP_HEADER_HEIGHT, DesktopHeader } from './DesktopHeader'
-import { calcMobileHeaderHeight, MobileHeader } from './MobileHeader'
+import { DesktopHeader } from './DesktopHeader'
+import { MobileHeader } from './MobileHeader'
 import { HeaderProps, NavigationSection } from './types'
 
+/* empty box to take the place behind the header (given it's sticky) */
```

Comment:
I was happy for a moment and thought you deleted it, but now it's back here. This is such an ugly hack.

Why not add `position: sticky` to both `DesktopHeader` and `MobileHeader` (in both cases on probably the `AppBar`) and delete this spacer. Works fine if I quickly try it with Inspect Element